conversions/hex-bin.py

```diff
@@ -0,0 +1,65 @@
+#  CONVERT HEXADECIMAL TO BINARY
+import math
+from typing import Union
+
+def convert(num: str) ->Union[bool, str]:
+    """
+    Convert a hexadecimal value to its decimal equivalent
+    #https://stackoverflow.com/questions/1425493/convert-hex-to-binary
+    Here, we have used " >> " bitwise operator
+    Bitwise right shift: 
+    Shifts the bits of the number to the right and fills 0 on voids left as a result. 
+    Similar effect as of dividing the number with some power of two.
+    Example: 
+    a = 10
+    a >> 1 = 5 
+    
+    >>> convert("AC")
+    10101100
+    >>> convert("9A4")
+    100110100100
+    >>> convert("   12f   ")
+    100101111
+    >>> convert("FfFf")
+    1111111111111111
+    >>> convert("F-f")
+    Traceback (most recent call last):
+    ...
+    ValueError: invalid literal for int() with base 16:
+    >>> convert()
+    Traceback (most recent call last):
+    ...
+    ValueError: Empty string was passed to the function:
+    
+    """
+
+    bin_str :str=""
+    flag :bool = 0
```

Comment:
```suggestion
```
Let's just use `is_negative` instead of creating a new `flag` variable.  The name `flag` is weak compared to `is_negative` because the former does not tell the reader what we are flagging where `is_negative` is self-documenting.  `bool` in Python should contain one of two values: `True` or `False` so assigning `0` will probably be rejected by mypy and will also confuse the reader.  Let's just drop `flag` and use `is_negative` instead.